201-ospatching-extension-on-ubuntu/azuredeploy.json

```diff
@@ -129,7 +161,7 @@
       "type": "Microsoft.Storage/storageAccounts",
       "name": "[parameters('newStorageAccountName')]",
       "apiVersion": "2015-05-01-preview",
```

Comment:
Can you make the api version a variable so it's easier top move to newer version headers?